### ISSUE TITLE
The windows 64 bit install scripts fail for me

### ISSUE THREAD

**Issue by PatrikLundell** [DFHack/dfhack#1190]
The first line of those scripts is
"call "C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\vcvarsall.bat" amd64", and the Visual Studio 14.0\VC directory doesn't contain the VC directory. There's a corresponding directory and file in 12.0, but changing the script to use that one caused another error further down the line.
In contrast, the first line of the 32 bit scripts is:
"call "%VS140COMNTOOLS%vsvars32.bat", and those scripts do work.

The underlying compilation works both for the 64 and 32 bit versions when done from within Visual Studio, but then the copying to the target DF directories becomes a manual task (or I've failed to figure out how to make Visual Studio do it).

DFHack and Visual Studio was downloaded as per the instructions on how to contribute.

**Comment by lethosor** [comment]
> the Visual Studio 14.0\VC directory doesn't contain the VC directory

It shouldn't. Was this a typo?

Also, yeah, you can't mix-and-match pieces from different versions of Visual Studio.

What if you try a few other paths? "Program Files" instead of "Program Files (x86)"?   What if you replace the first line with
```
call "%VS140COMNTOOLS%vcvarsall.bat" amd64
```
?

**Comment by lethosor** [comment]
[This thread](https://social.msdn.microsoft.com/Forums/en-US/1071be0e-2a46-4c30-9546-ea9d7c4755fa/where-is-vcvarsallbat-file?forum=visualstudiogeneral) might also help (although it's a bit hard to understand).

**Comment by PatrikLundell** [comment]
Yes, it's a typo. I originally referred to the non existent file, and then found the directory was missing as well, so it should have said "the Visual Studio 14.0 directory doesn't contain the VC directory". The only directory present in 14.0 is XML.
I've tried your proposed version as well, and it fails to find the bat file. As mentioned, I can work around it with some transpiration, but I thought others may be subjected to it as well (I did work around it a couple of months ago when testing my plugin, but didn't write an issue at that time).

**Comment by PatrikLundell** [comment]
Looks like you nailed it directly. The link isn't too hard to understand (despite the incorrect double negations). Given both that it worked for 32 bit and that I did a FULL install after some other things were missing, it never occurred to me that required things would be left out.
(And I still haven't learned that Close and Comment does NOT mean "close this window and post the comment").
There's something wrong with my computer, it seems. I've manually installed the extra package 3 times, only to find nothing has been added and the installation checkbox is unchecked. Looks lime lengthy maintenance is required, as it seem to take forever to run the "repair" option.
